docs/content/components/radio.mdx

```diff
@@ -21,90 +32,50 @@ import { Radio } from '@marigold/components';
 
 ## Props
 
-| Property                  | Type      | Default     |
-| :------------------------ | :-------- | :---------- |
-| `id`                      | `string`  |             |
-| `variant` (optional)      | `string`  | `__default` |
-| `labelVariant` (optional) | `string`  | `inline`    |
-| `required` (optional)     | `boolean` | `false`     |
-| `disabled` (optional)     | `boolean` | `false`     |
-| `error` (optional)        | `boolean` | `false`     |
-| `errorMessage` (optional) | `string`  |             |
+| Name               | Type      | Default     | Description                                                                                                        |
+| :----------------- | :-------- | :---------- | :----------------------------------------------------------------------------------------------------------------- |
+| `variant`          | `string`  |             | Use a different _variant_ from theme                                                                               |
+| `size`             | `string`  | `'level-1'` | Use a different _size_ from theme                                                                                  |
+| `error` (optional) | `boolean` | `false`     | If `true`, the checkbox is considered invalid and if set the `errorMessage` is shown instead of the `description`. |
+| ...                |           |             | Yes you can use all regular attributes of `input`!                                                                 |
 
 ## Examples
```

Comment:
Examples for description and errorMessage?

Reply:
Not supported yet, figma has no designs for this. Also, not supported by the `react-aria` hooks :-/